README.md

```diff
@@ -75,3 +75,17 @@ This project supports hot reloading of tests and the demo page.
 * **All component properties should be documented.** Property documentation should link to relevant Rivet documentation if available, particularly if the property is for styling and presentation options. Look to existing components for examples.
 * **Components should be included in the style guide.** `styleguide.config.js` organizes component documentation into sections that align with the Rivet component documentation. If you're adding a new component, please place its documentation in the appropriate section.
 * **Components should have usage examples.** Every component should have a markdown file (`.md`) alongside its implementation file (`.tsx`) with usage examples for the style guide. Look to existing components for examples.
+
+### Semantic Release
```

Comment:
This information seems like it belongs better in a `CONTRIBUTING.md` or `DEVELOPERS.md` file (honestly, I think a lot of the README does, but we can deal with that later). In this case, it would be helpful to have this information (especially the Angular Commit Message information) linked from pull requests.